pages/update.js

```diff
@@ -42,6 +45,15 @@ class UpdatePage extends React.Component {
 
   state = { isReloadingData: false };
 
+  componentDidMount() {
+    const { router, data, updateSlug } = this.props;
+    const query = router.query;
+    const account = data?.account;
+    if ([CollectiveType.EVENT, CollectiveType.PROJECT].includes(account?.type) && !query.parentCollectiveSlug) {
+      router.push(`${getCollectivePageRoute(account)}/updates/${updateSlug}`, undefined, { shallow: true });
+    }
+  }
```

Comment:
Could we have a common helper for this since we're repeating it in multiple components? Something like:

```suggestion
  componentDidMount() {
    addParentToURLIfMissing(router, account);
  }
```

Also, what about cases where `account` is not loaded yet? We're going to redirect to `/collective/updates/${updateSlug}`, without including the real parent slug. Would be great to handle this properly.

Reply:
Thanks. I've extracted this to a common helper. 👍🏽